prometheus/counter.go

```diff
@@ -35,6 +35,9 @@ type Counter interface {
 	// Prometheus metric. Do not use it for regular handling of a
 	// Prometheus counter (as it can be used to break the contract of
 	// monotonically increasing values).
+	// This method is DEPRECATED. Use NewConstMetric to create a counter for
```

Comment:
Thanks for the pointer. I'll fix it where it is "real" deprecation. Situation with Handler and friends is special, but I'll have another trick up my sleeves there in the next commit. Stay tuned…